src/assets/javascripts/_/index.ts

```diff
@@ -31,7 +31,10 @@ import { Version } from "~/templates/version"
  * Feature flag
  */
 export type Flag =
+  | "announce.dismiss"                 /* Dismissable announcement bar */
```

Comment:
are any of these other features working / should be documented?

Reply:
I haven't tested them yet. I just figured it would hurt to have them while we figure it out.

Reply:
yeah sounds reasonable.

Reply:
`announce.dismiss` works well. I added a banner to our docs that declares the latest upstream version used (with a link to the upstream changelog). I also used the new HTML template as an example in the "New Blocks" section of the customization page. The `announce.dismiss` feature is enabled should we need to verify that it behaves consistently.

---
I'm not sure how to test the `content.lazy` feature since it is browser specific. So, I'm just leaving that flag in there in case we want to re-assess later.